Packet++/header/Layer.h

```diff
@@ -5,6 +5,7 @@
 #include <stdio.h>
 #include "ProtocolType.h"
 #include <string>
+#include <sstream> // VS removed the iostreams stuff from string in VS2019
```

Comment:
Why is this change needed? We didn't change anything in this file in this PR... 🤔

Reply:
I'm not sure why the changes affect the whole code but when I remove this line it gives error on the other layers. I will share the compile log for Windows. Maybe, you will have an idea

Reply:
Looks like I found the solution. I should include `sstream` before the `Layer.h` at `SingleCommandTextProtocol.h`